### ISSUE TITLE
check/correct metrics calculation/output

### ISSUE THREAD

**Issue by anasedova** [knodle/knodle#221]
1) what metrics should be logged when testing on dev set
2) what metrics should be logged when testing on test set
3) the same metric by testing on the dev set: 
- if sklearn evaluation: train set - acc, dev set - acc, test set - all metrics
- if ex-tac evaluation with other class: train set - f1, dev set - f1, test set - all metrics (without acc)
(having one single evaluation metric everywhere (within one experiment), regardless of if we are in train or in test phase)
4) log all metrics (prec, recall, f1) on testing and only one (acc/f1) on train/dev? or print out the whole report everywhere?
